### ISSUE TITLE
[follow_line] Incorrect initial gazebo view

### ISSUE THREAD

**Issue by pariaspe** [JdeRobot/RoboticsAcademy#1914]
Change the initial gazebo view position to a better one. It can be done by adding these lines to the existing worlds:
```xml
<gui fullscreen='0'>
    <camera name='user_camera'>
        <pose frame=''>x y z R P Y</pose>
        <view_controller>orbit</view_controller>
        <projection_type>perspective</projection_type>
    </camera>
</gui>
```

**Comment by sameergupta4873** [comment]
@pariaspe I think this is pretty straight forward, can I work on this ?

**Comment by MashiatK** [comment]
I want to work on this. If the issue is open, please assign it to me.

**Comment by jmplaza** [comment]
Hi @MashiatK , there is no need for explicit assignment, just work on it and feel free to submit a Pull Request solving it :-)

**Comment by GauravKumar9920** [comment]
Hey @pariaspe, I am updating the initial view location of the world with following views -
for Default line - 
![default-1](https://user-images.githubusercontent.com/84905312/219852444-c167364e-2e6b-408a-833f-4e843f7d9bc5.png)
for montmelo -
![montmelo-1](https://user-images.githubusercontent.com/84905312/219852457-52ca406e-d437-44d5-a28a-427dc8922e25.png)
for montreal - 
![montreal-1](https://user-images.githubusercontent.com/84905312/219852466-94938554-5903-498a-91f7-7285957ba6c8.png)
and the last one N..-
![N-1](https://user-images.githubusercontent.com/84905312/219852483-c172e877-2860-4998-822d-401e0a0e5554.png)

if you suggest any changes do lemme know.

**Comment by pariaspe** [comment]
Hi, all of them look fine to me!

**Comment by pariaspe** [comment]
Solved in #1936 , thanks @GauravKumar9920 !
